### PR TITLE
Jacobi iterative b1

### PR DESCRIPTION
**Pull request by ShwethaSelma**
# Adding a New Sample(s)

## Description

Jacobi Application Migration from CUDA to SYCL

## Checklist
Administrative
- [ x] Review sample design with the appropriate [Domain Expert](https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts): <Krishnappa, Rakshith>

Code Development
- [ x] Implement coding guidelines and ensure code quality. [see wiki for details](https://github.com/oneapi-src/oneAPI-samples/wiki/General-Code-Guidelines)
- [x ] Adhere to readme template 
- [x ] Enforce format via clang-format config file
- [x ] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [x ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [ x] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com

Security and Legal
- [ ] OSPDT Approval (see Project Manager for assistance)
- [ x] Compile using the following compiler flags and fix any warnings, the falgs are: "/Wall -Wformat-security -Werror=format-security"
- [ ] Bandit Scans (Python only)
- [ ] Virus scan

Review
- [x] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [x ] Review readme with Tom Lenth(@tomlenth) and/or Project Manager
- [x] Tested using Dev Cloud when applicable

### CODE REVIEW

**Review by praveenkk123 — DISMISSED** [review]
Approve for CI

**Review by praveenkk123 — DISMISSED** [review]
Approve for CI

**Review by praveenkk123 — DISMISSED** [review]

File: DirectProgramming/DPC++/DenseLinearAlgebra/jacobi_iterative/sycl_dpct_migrated/src/jacobi.h [hunk 1]
```diff
@@ -0,0 +1,12 @@
+//=========================================================
+// Copyright © 2020 Intel Corporation
```
Comment (outoftardis) [inline DirectProgramming/DPC++/DenseLinearAlgebra/jacobi_iterative/sycl_dpct_migrated/src/jacobi.h:2]: shouldn't it be 2022? same question for other files

```suggestion
// Copyright © 2022 Intel Corporation
```

**Review by tomlenth — APPROVED** [review]
Thanks for the detailed update to the readme!

**Review by praveenkk123 — APPROVED** [review]

**Pull request merged**